x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_requirements_page/components/fleet_server_on_prem_instructions.tsx

```diff
@@ -1,752 +0,0 @@
-/*
```

Comment:
The contents of this file have been replaced with distinct components in the `fleet/components/fleet_server_instructions` files above.